### DISCUSSION TITLE
Autocomplete variable limits

### DISCUSSION THREAD

**Discussion by jberpr** [grafana/grafana#46993] [Dashboards, Panels & Visualizations]
Hello,
I am finding a strange behavior  with the variables limit, it seems that one dashboard has a 1000 entries limit. Other dashboard has a 1150 limit, does this respond to anything? As I have seen on another discussions, the default limit should be 1000 entries for each variable, could it be surpassed?

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
